crazy_rl/test/test_jax.py

```diff
@@ -0,0 +1,419 @@
+import jax.numpy as jnp
+import jax.random as random
+
+from crazy_rl.multi_agent.jax.catch.catch import Catch
+from crazy_rl.multi_agent.jax.circle.circle import Circle
+from crazy_rl.multi_agent.jax.escort.escort import Escort
+from crazy_rl.multi_agent.jax.hover.hover import Hover
+from crazy_rl.multi_agent.jax.surround.surround import Surround
+
+
+def two_drones_crash(parallel_env, key):
+    """Test where two drones starting on (0, 0, 1), (0, 1, 1) crash together."""
+    state = parallel_env.reset(key)
+
+    actions = jnp.array([[0, 1, 0], [0, -1, 0]])
```

Comment:
make a function `move` that defines the action and does the step, this will remove a lot of duplication